tools/llm_bench/requirements_2024.5.txt

```
@@ -0,0 +1,21 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+numpy==2.0.2
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+auto_gptq==0.7.1
+pillow==11.0.0
+torch==2.5.1+cpu
+transformers==4.46.2
+diffusers==0.31.0
+openvino==2024.5.0
+#optimum is in dependency list of optimum-intel
+git+https://github.com/huggingface/optimum-intel.git@d35737671ec226a66c495159588b7603066e1b07#egg=optimum-intel
+nncf ~= 2.14.0
+packaging==24.1
+psutil==6.1.0
+timm==1.0.11
+tiktoken==0.8.0
+onnx==1.16.1
```

for avoiding issues with dependabot
```suggestion
onnx==1.17.0
```